### PR TITLE
[WiP] Rename Matrix variables and reorder parameter input

### PR DESCRIPTION
**Pull request by Joott**
#43

### CODE REVIEW

File: src/FSharp.Stats/Matrix.fs [hunk 1]
```diff
@@ -15,220 +15,223 @@ open System.Diagnostics
...
-        
+
     module Generic = begin
 
         module MS = SpecializedGenericImpl
 
         // Accessors
-        let get (a:Matrix<_>) i j   = a.[i,j]
-        let set (a:Matrix<_>) i j x = a.[i,j] <- x              
+        let get (matrix:Matrix<_>) index1 index2 = matrix.[index1,index2]
```
Comment (bvenn) [inline src/FSharp.Stats/Matrix.fs]: change index1 and index2 to indexCol and indexRow

File: src/FSharp.Stats/AlgTypes.fs [hunk 2]
```diff
@@ -285,6 +285,23 @@ namespace FSharp.Stats
               | _ -> raise (new System.NotSupportedException("The element type '"+(typeof<'T>.ToString())+"' carried by this vector or matrix does not support the INormFloat<_> operation (i.e. does not have a registered numeric association that supports this type)"))
 
         let mkDenseMatrixGU ops arr = DenseMatrix(ops,arr)
+        let mkSparseMatrixGU ops (arr: 'a[,]) =
+            let m = arr |> Array2D.length1
```
Comment (bvenn) [inline src/FSharp.Stats/AlgTypes.fs]: please change variable names to more descriptive ones

Comment (Joott) [reply src/FSharp.Stats/AlgTypes.fs]: Changed variable names

File: src/FSharp.Stats/Matrix.fs (hunk 1 above)
Comment (Joott) [reply src/FSharp.Stats/Matrix.fs]: Changed variables index_ and length_ to _Row and _Col

**Pull request merged**